src/digraph.rs

```diff
@@ -179,6 +179,24 @@ impl NodeCount for PyDiGraph {
 
 // Rust side only PyDiGraph methods
 impl PyDiGraph {
+    fn _add_edge_no_checks(
```

Comment:
```suggestion
    fn add_edge_no_cycle_check(
```
since we're still doing a check if a graph is not a multigraph and in Rust the underscore usually means unused variable/function.